NEWS.md

```diff
@@ -1,6 +1,7 @@
 # bslib (development version)
 
 * `navset_card_pills()`, `navset_card_underline()`, `navset_card_tabs()` fixed to now respect header/footer arguments (@tanho63, #1024) 
+* added terminating semicolons to declarations in `value_box.scss` and `sidebar.scss` to prevent issues with parsers that require them. (@cscheid, #1104)
```

Comment:
```suggestion
* Added terminating semicolons to declarations in `value_box.scss` and `sidebar.scss` to prevent issues with parsers that require them. (@cscheid, #1104)
```

Reply:
No offense @cscheid but I don't think this is a newsworthy change :smile

Reply:
I didn't know your criteria, my bad!